db/migrate/20170107034803_create_orders.rb

```diff
@@ -0,0 +1,11 @@
+class CreateOrders < ActiveRecord::Migration[5.0]
+  def change
+    create_table :orders do |t|
+      t.integer :total_price_in_cents
+      t.references :status, index: true, foreign_key: true
```

Comment:
Thanks for catching this!  I've added some model specs, and this information is set before the order object gets saved.